packages/suite/src/views/wallet/anonymize/components/TooltipIcon.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import { Translation } from '@suite-components';
+import { ExtendedMessageDescriptor } from '@suite-types';
+import { Icon, Tooltip } from '@trezor/components';
+
+const StyledIcon = styled(Icon)`
+    margin-left: 6px;
+`;
+
+interface TooltipIconProps {
+    message: ExtendedMessageDescriptor['id'];
+}
+
+export const TooltipIcon = ({ message }: TooltipIconProps) => (
```

Comment:
There's a `TooltipSymbol` component already, I'd use that. In the research doc it was mentioned that some people found the icons a bit too distracting, so it already looks more fitting :D

Reply:
Good point. I actually prefer the newer design but let's unify it. I simplified the `TooltipSymbol` component a little bit and made the icon a tad bigger for my old eyes to see it. I also made use of the `TooltipButton` when I was at it.

Reply:
Looks cleaner! We can figure out the icon later, since I feel it might be changes altogether. Or if you want, make the `TooltipSymbol` accept an icon to use :D